.gitignore

```diff
@@ -1,4 +1,4 @@
+bits/
```

Comment:
Perhaps anchor these to the root dir via:

```
/bits
/build
...
```

Not a biggie. There are unlikely to be "bits" or "build" dirs in subdirs that one wants commited to the repo.